gini/gini-vision-lib-ios
# Remove document validation on review screen

Some documents are not reviewable, so they do not have to go through the review screen. On the Component API 

### How to test
Import a PDF and see that it doesn't go through the review screen. If instead you import an image or take a picture with the camera, it should be reviewable from review screen. 
Also using the "Open with" method, you should have the same result.

### Merging
Automatic.

### Ticket 
Issue #64
